test/dotcom/system_status_test.exs

```diff
@@ -1,21 +1,70 @@
-defmodule Dotcom.SystemStatus.AlertsTest do
-  use ExUnit.Case, async: true
-  doctest Dotcom.SystemStatus.Alerts
+defmodule Dotcom.SystemStatusTest do
+  use ExUnit.Case
 
+  import Dotcom.Alerts, only: [service_impacting_effects: 0]
+  import Dotcom.SystemStatus
+  import Dotcom.Utils.ServiceDateTime, only: [service_range_day: 0]
+  import Mox
   import Test.Support.Factories.Alerts.Alert
 
-  alias Dotcom.SystemStatus.Alerts
+  setup :verify_on_exit!
 
-  describe "active_on_day?/2" do
+  setup _ do
+    stub_with(Dotcom.Utils.DateTime.Mock, Dotcom.Utils.DateTime)
+    :ok
+  end
+
+  describe "subway_status/0" do
+    test "requests alerts for all subway lines @ today datetime" do
+      today = Test.Support.Generators.DateTime.random_date_time()
+
+      expect(Dotcom.Utils.DateTime.Mock, :now, 2, fn ->
+        today
+      end)
+
+      expect(Alerts.Repo.Mock, :by_route_ids, fn route_ids, datetime ->
+        assert Enum.sort(route_ids) == Dotcom.Routes.subway_route_ids() |> Enum.sort()
+        assert datetime == today
+
+        []
+      end)
+
+      _ = subway_status()
+    end
+
+    test "returns status without alerts ending earlier that day" do
+      route_id_with_alerts = Dotcom.Routes.subway_route_ids() |> Faker.Util.pick()
+      line = Dotcom.Routes.line_name_for_subway_route(route_id_with_alerts)
+      {day_start, day_end} = service_range_day()
+      earlier_end = Timex.shift(day_start, minutes: 1)
+      alert_today = disruption_alert({day_start, day_end}, route_id_with_alerts)
+      alert_earlier = disruption_alert({day_start, earlier_end}, route_id_with_alerts)
```

Comment:
It took me a bit to understand what these variables were. What about names like 👇?

```suggestion
      currently_active_alert = disruption_alert({day_start, day_end}, route_id_with_alerts)
      expired_alert = disruption_alert({day_start, earlier_end}, route_id_with_alerts)
```

Reply:
I'll make this change